Projects/Features/Scene/ShortStorageScene/ShortStorageNewsList/ShortStorageNewsListView.swift

```diff
@@ -169,3 +177,89 @@ private struct EmptyNewsContentView: View {
     }
   }
 }
+
+// MARK: - Tooltip View
+private struct TooltipView: View {
+  private let store: Store<ShortStorageNewsListState, ShortStorageNewsListAction>
+  
+  fileprivate init(store: Store<ShortStorageNewsListState, ShortStorageNewsListAction>) {
```

Comment:
누나 ~ TooltipView에서 `ShortStorageNewsListState`는 사용을 안 하는 것 같으니 아래처럼 바꾸면 오똔까요

```suggestion
  TooltipView(store: store.stateless)
  
  private struct TooltipView: View {
    private let store: Store<Void, ShortStorageNewsListAction>
  
    fileprivate init(store: Store<Void, ShortStorageNewsListAction>) { }
  }
```

Reply:
오 이건 생각을 못해봤네. 
근데 나중에라도 사용할 수 있으니까 남겨놓는 것도 방법이지 않을까 명확하게? 일단 Void로 변경해보려는데, 의견이 궁금해! 지금 사용하지 않으니까 들고있지 않는게 더 좋을까??

-> 라고 하고 변경하려고 했는데, TooltipView init할 때 store넘겨주는데, 그 때 그러면 타입이 달라져서 새로 Void랑 액션을 가진 Store를 만들어서 넘겨줘야하는 상황이 발생함뮈다?!🤔
오히려 그냥 두는게 더 낫지 않나 라는 생각이 드네, 실제로 지금 저 TooltipView은 차후에라도 State로 ShortStorageNewsListState만 받게 될 거 같은데🤔
일단 변경 없이 머지해두고 추가 의견 주시면 차후 PR에 반영할게여~